tests/benchdnn/graph/utils.hpp

```diff
@@ -77,8 +77,9 @@ enum class dnnl_driver_t {
 };
 
 enum class graph_recognized_pattern_t {
-    sdpa,
     ordinary,
+    sdpa_fwd,
```

Comment:
Just to confirm, it's intended that we don't differentiate forward and inference here, right?

Reply:
yes, correct